File: mario.c
// Include libraries
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height, row, column, space;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    
    // Display the number of rows
    for (row = 0; row < height; row++)
    {
        // Display the number of spaces
        for (space = 0; space < height - row - 1; space++)
        {
            printf(" ");
        }
        // Display the number of #s with the columns
        for (column = 0; column <= row; column++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
# Input validation: The height has to be greater than 0 and less than 9
from cs50 import get_int
while True:
    try:
       height = int(input("Height: "))
       if height >= 1 and height <= 8:
        break
       else:
        print("", end="")
# The number of spaces accounted
spaces = 1
# Account index for rows and count for columns
for count in range(height):
    for spaces in range(height - count - 1):
        print("", end="")
    for index in range(count + 1):
        print("#", end="")
    print()





